docs/deploying.rst

```diff
@@ -0,0 +1,33 @@
+Deploying
+=========
+
+Make sure you are the only one deploying
+----------------------------------------
+
+There should be a way to make sure that only one person is trying to change a server at a time.
+
+Currently we use a wiki page that only Open Data Services Coop have access to. This means only Open Data Services Coop should deploy to live servers.
+
+TODO - Work out how we're going to make a deploy token work between Open Data Services and OCP, so that OCP staff can deploy. Document.
+
+Make sure you have the latest scripts
+-------------------------------------
+
+You can run
+
+.. code-block:: bash
+
+    ./updateToMaster.sh
+
+This will update all 3 repositories (public and 2 private ones) to the master branch and the latest version,
```

Comment:
Link to where one can find details of all the repos?

Reply:
I'm not sure why the person would want to know that on this page. It's listed on a previous page, which I think is the point they would care about that.